MAINTAINERS.md

```diff
@@ -1,21 +1,29 @@
 ## Maintainers
 
-| Name | GitHub Id | email |
-|---|---|---|
-| Makoto Takemiya | takemiyamakoto | takemiya@soramitsu.co.jp |
-| Ryu Okada | ryuo88 | okada@soramitsu.co.jp |
-| Taisei Igarashi | MizukiSonoko | igarashi@soramitsu.co.jp |
-| Motohiko Abe | motxx | abe@soramitsu.co.jp |
-| Daisuke Shimada | cimadai | dice.k1984@gmail.com |
-| Sushant D. Mayekar | Sushantdm |  Sushantdm@gmail.com |
-| Yanno Ban | yannoban | ban.yanno@nbc.org.kh
-| Hiroshi Sasagawa | SasagawaHiroshi | sasagawa_hiroshi@intec.co.jp |
-| Takumi Yamashita | satelliteyes | yamashita@soramitsu.co.jp |
-| Bogdan Vaneev | Warchant | bogdan@soramitsu.co.jp |
-| Fyodor Muratov | muratovv | fyodor@soramitsu.co.jp |
-| Andrei Lebedev | lebdron | andrei@soramitsu.co.jp |
-| Bulat Nasrulin | grimadas | bulat@soramitsu.co.jp |
-| Kamil Salakhiev | kamilsa | kamil@soramitsu.co.jp |
-| Konstantin Munichev | luckychess | konstantin@soramitsu.co.jp |
-| Evgenii Mininbaev | l4l | evgenii@soramitsu.co.jp |
-| Nikolay Yushkevich | neewy | nikolai@soramitsu.co.jp |
+Maintainers of Hyperledger Iroha project
+are supposed to help contributors by explain them project details,
+such as architecture, process, existing issues.
+
+This is the list of maintainers, including their email address for direct communications:
+
+|          Name          |     GitHub Id            |               email            |         Area of expertise       |
+|------------------------|--------------------------|--------------------------------|---------------------------------|
+| Makoto Takemiya        | takemiyamakoto           | takemiya@soramitsu.co.jp       | Product vision                  |
```

Comment:
maybe prepend `@` to the github id-s, so it will be clickable?

Reply:
@l4l 

Am I doing something wrong?

![screen shot 2018-04-12 at 14 10 30](https://user-images.githubusercontent.com/11841667/38673901-5354352c-3e5b-11e8-858a-3af370af44c4.png)

Reply:
It is not clickable

Reply:
:(